example/dist/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This file should live in the `example/src/` folder because we need to ignore the `dist/` folder completely. We should find a way to copy the file into `dist/` upon build instead. With WebPack, we can usually achieve that using [HtmlWebpackPlugin](https://webpack.js.org/plugins/html-webpack-plugin/) or [CopyWebpackPlugin](https://webpack.js.org/plugins/copy-webpack-plugin/), perhaps there's something similar for metro 🤔